packages/context/src/interceptor.ts

```diff
@@ -284,13 +286,16 @@ export function intercept(...interceptorOrKeys: InterceptorOrKey[]) {
  * @param target - Target class (for static methods) or object (for instance methods)
  * @param methodName - Method name
  * @param args - An array of argument values
+ * @param options - Options for the invocation
  */
 export function invokeMethodWithInterceptors(
   context: Context,
   target: object,
   methodName: string,
   args: InvocationArgs,
+  options: InvocationOptions = {},
 ): ValueOrPromise<InvocationResult> {
+  assert(!options.skipInterceptors, 'skipInterceptors is not allowed');
```

Comment:
What is the reasoning behind the decision to disallow `skipInterceptors`? What is that flag good for when it cannot be used? Please add code comments to explain.

Reply:
The function name is `invokeMethodWithInterceptors`. That's why I added an assertion to make sure `skipInterceptors` cannot be set to `true`.